snap-template/snapcraft.yaml

```diff
@@ -0,0 +1,30 @@
+name: my-snap  # the name of the snap
+version: 0  # the version of the snap
+summary: This is my-snap's summary  # 79 char long summary
+description: |
```

Comment:
otherwise, it's not going to be a valid yaml file (as per spec), I find is as well cleaner to have then all the text aligned